### PR TITLE
Add john sj

### PR DESCRIPTION
**Pull request by genehack**
I no longer use Twitter, but I am fairly active on the Fediverse, so I updated the member layout to support having a `fediverse` object in the YAML front matter, with server and handle sub-keys (since different `fediverse` servers have different base URLs, unlike the constant `https://twitter.com` or `https://github.com`). I'm not sure if there's some documentation of supported entries in the team section front matter that I should also update; please point me at it if there is.

As I explained in the commit message in that change, I picked a somewhat arbitrary FA icon to use for that section; open to suggestions for a better one.

Also completely open to a decision that this is not something y'all want to support, in which case I'll send in a new PR sans fediverse stuff.

### CODE REVIEW

File: team/_posts/2024-04-29-john-sj-anderson.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+handle: genehack
+github: genehack
+fediverse:
+  server: dementedandsadbut.social
+  handle: "@genehack"
+image: /images/team/john-sj-anderson.jpg
+website: https://genehack.org
+---
+
+I joined the Bedford Lab near the end of 2024. I have a PhD in
```
Comment (victorlin) [inline team/_posts/2024-04-29-john-sj-anderson.md:14]: > near the end of 2024

Hmm, you're a bit ahead of schedule 😄

Comment (genehack) [reply team/_posts/2024-04-29-john-sj-anderson.md]: 🤦 good catch — updated with "end of April, 2024"

File: _layouts/member.html [hunk 2]
```diff
@@ -47,6 +47,17 @@
...
+		<div class="smallhead">
+			Fediverse
+		</div>
+		<div class="pad-left note">
+			<div class="smallspacer"></div>
+			<i class="fa fa-handshake-o"></i>
+			<a class="off" href="https://{{ member.fediverse.server }}/{{ member.fediverse.handle }}">{{ member.fediverse.handle }}</a>
+		</div>
+		<div class="bigspacer"></div>
+		{% endif %}
```
Comment (victorlin) [inline _layouts/member.html:60]: > Also completely open to a decision that this is not something y'all want to support, in which case I'll send in a new PR sans fediverse stuff.

I don't have a problem with this - it doesn't affect any other member pages and looks fine on your page.

> I'm not sure if there's some documentation of supported entries in the team section front matter that I should also update; please point me at it if there is.

Not that I know of. I'm assuming most people have just looked at existing pages for inspiration.

**Review by victorlin — APPROVED** [review]
Built and viewed locally - looks good!

**Comment by trvrb** [comment]
I appreciate the `fediverse` extension. Thank you @genehack! I've deployed update to bedford.io.

**Pull request merged**